### DISCUSSION TITLE
build with Vite not working

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#15427] [CLI - PWA mode]
I'm working on a large complex app. Converted from Vuex to Pinia, so also switched to use Vite instead of webpack.
Works fine in development mode, but when I build it, the resulting page is blank and shows several errors like this:
   index.a03a000d.js:1          Failed to load resource: the server responded with a status of 404 (Not Found)
Also seeing:
WebSocket connection to 'ws://127.0.0.1:5500//ws' failed: 

To try and understand what I'm doing wrong, I scaffolded a new project with quasar cli and tried to build the example app that comes with Quasar. But I get a similar result...
  index.99652668.css:1          Failed to load resource: the server responded with a status of 404 (Not Found)
  index.9065343d.js:1          Failed to load resource: the server responded with a status of 404 (Not Found)

Any suggestions will be appreciated!

**Comment by yusufkandemir** [comment]
I also answered a similar question of yours at https://github.com/quasarframework/quasar/issues/15437#issuecomment-1428504744. After also seeing this, I can confidently say something is wrong with your web server configuration and/or some web extensions or other applications are messing up the stuff.

**Comment by CrawfordW** [comment]
Yusuf,
Thanks. You're right - it works OK if I use quasar serve.
Any suggestions why my XAMPP server works when I build with webpack but 
not with Vite?
I don't know about server configurations.


------ Original Message ------
From "Yusuf Kandemir" ***@***.***>
To "quasarframework/quasar" ***@***.***>
Cc "Crawford Woodman" ***@***.***>; "Author" 
***@***.***>
Date 2/13/2023 2:15:42 PM
Subject Re: [quasarframework/quasar] build with Vite not working 
(Discussion #15427)

**Reply by CrawfordW** [reply]
Still wrestling with this. I'm using apache server with XAMPP because I also need mySQL and PHP support. Everything works fine EXCEPT the quasar  app built with Vite. The same source code built with Webpack works fine when built. I accept that this could be a server configuration issue, but cannot find anything that helps.

**Reply by CrawfordW** [reply]
Get the same problem when I upload to my real host provider...
Getting 404 error (Not Found) on these two files, even though they are present:
assets/index.63ea4fab.css
assets/index.a881ef79.js

**Reply by s-light** [reply]
late but maybe it helps -
do you have access to your apache logs?
there you could check what apache things about these files..
can you open them manually from a webbrowser?
i think that there is some permission or other path mapping thing going on that messes up the sub-path...

maybe webpack had some other output directory structure..

**Reply by dannykhoo** [reply]
> late but maybe it helps - do you have access to your apache logs? there you could check what apache things about these files.. can you open them manually from a webbrowser? i think that there is some permission or other path mapping thing going on that messes up the sub-path...
> 
> maybe webpack had some other output directory structure..

i have facing the same problem , old quasar projects still working fine , however recently i have created new quasar project using vite, can't even open them manually from webbrowser.

![image](https://github.com/user-attachments/assets/3db71e1c-aedd-4aea-ab43-aaf4335a6808)
